src/pretalx/mail/models.py

```diff
@@ -309,22 +316,27 @@ def send(self, requestor=None, orga: bool = True):
         to = self.to.split(",") if self.to else []
         if self.id:
             to += [user.email for user in self.to_users.all()]
-        mail_send_task.apply_async(
-            kwargs={
-                "to": to,
-                "subject": self.prefixed_subject,
-                "body": text,
-                "html": body_html,
-                "reply_to": (self.reply_to or "").split(","),
-                "event": self.event.pk if has_event else None,
-                "cc": (self.cc or "").split(","),
-                "bcc": (self.bcc or "").split(","),
-                "attachments": self.attachments,
-            },
-            ignore_result=True,
-        )
-
-        self.sent = now()
+            queuedmail_pre_send.send(
```

Comment:
Let’s move the signal call above `text = self.make_text()`, so that the signal consumers can actually change the content of the email being sent, I think that would be nice.